src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0; // Змінна для збереження поточної суми
+
+  function adder(...args) {
```

Comment:
According to the task requirements, the `adder` function should accept only one argument. But here you are allowing it to accept multiple arguments using the rest parameter syntax (`...args`). This might lead to incorrect function behavior.